README.md

```diff
@@ -84,7 +84,7 @@ services:
         deploy:
           resources:
             limits:
-              memory: 16G
+              memory: 5G
```

Comment:
I would suggest setting the reservation to 5 and leaving the limit at 16.  This will not deploy as the limit is lower than the reservation.

Reply:
1. Yes you're right must have forgotten this...
2. I wouldn't leave the imit by 16

I ran some tests:
On a mid game safe game the server consumes between 2 and 4GB of RAM in endgame its like 4 to 5GB.
I'll update the numbers to this but it would be great if someone else could also test this to backup (or not refute) the numbers.